.github/workflows/release-stable.yaml

```diff
@@ -2,7 +2,7 @@ name: release-stable
 
 on:
   push:
-    tags: ["v[1-9]+.[0-9]+.[0-9]+"]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

Comment:
Weird, how did this work before? :D

Reply:
It did not 🪄 

I had to do it by hand 😢

Reply:
either that or we pushed to stable and that picked it up

What I was doing was to generate a tag + also push to stable and that would pick it up